webboot/webboot.go

```diff
@@ -148,15 +132,25 @@ func main() {
 
 	//Processes the URL to receive an io.ReadCloser, which holds the content of the downloaded file
 	log.Println("Retrieving the file...")
-	read, err := linkOpen(URL)
+	iso, err := linkOpen(URL)
 	if err != nil {
 		log.Fatal(err)
 	}
 
-	err = write(read, filename)
+	// TODO: Find a persistent memory device large enough to store the content. If no blocks are available, error the user.
+	pmem, err := os.OpenFile("/dev/pmem0", os.O_APPEND|os.O_WRONLY, 0600)
 	if err != nil {
 		log.Fatal(err)
 	}
+	if _, err := io.Copy(pmem, iso); err != nil {
+		log.Fatalf("Error copying to persistent memory device: %v", err)
+	}
+	if err = iso.Close(); err != nil {
+		log.Fatalf("Error closing reader for %v: %v", URL, err)
+	}
+	if err = pmem.Close(); err != nil {
+		log.Fatalf("Error closing /dev/pmem0: %v", err)
```

Comment:
is it okay for me to error the user out letting them know that the error occurred closing /dev/pmem0 such as this?

Reply:
yes. A close error can mean that there was an IO error of some sort. This is fine.

Reply:
Okay, sounds good :) thank you